infra/helper.py

```diff
@@ -595,10 +595,16 @@ def check_build(args):
       not _check_fuzzer_exists(args.project_name, args.fuzzer_name)):
     return 1
 
+  fuzzing_language = _get_project_language(args.project_name)
+  if fuzzing_language is None:
+    print('WARNING: language not specified in project.yaml. Defaulting to C++.')
```

Comment:
Is this branch needed, i thought Max forced it to always exist.

Reply:
No idea, but we do it anyway here: https://github.com/google/oss-fuzz/blob/master/infra/helper.py#L496

Reply:
sgtm